modules/ROOT/partials/configuration/tinycomments_can_edit_comment.adoc

```diff
@@ -20,7 +20,8 @@ The following example extends the default behavior to allow the author `<Admin u
 
```

Comment:
This option needs an id added for the heading at the top of the page. e.g.
```
[[tinycomments_can_edit_comment]]
=== `tinycomments_can_edit_comment`
```

Reply:
Done